### ISSUE TITLE
deploy node errors out deploy

### ISSUE THREAD

**Issue by chaffeqa** [getsentry/sentry-webpack-plugin#213]
```Creating an optimized production build...
[Sentry Webpack Plugin] DRY Run Mode
[Sentry Webpack Plugin] Creating new release:
 'unknown'
[Sentry Webpack Plugin] Calling upload-sourcemaps with:
 {
  debug: false,
  finalize: true,
  rewrite: true,
  include: [ '/home/runner/work/fantasy-picks/fantasy-picks/build' ],
  ignore: [ 'public/debug', 'public/static/ads-meta.js' ],
  stripPrefix: [ '/app/build', '/app' ],
  validate: true,
  dryRun: true,
  release: 'unknown',
  deploy: { env: 'local' }
}
[Sentry Webpack Plugin] Finalizing release:
 'unknown'
```

generates error:

```
Failed to compile.

(undefined) Sentry CLI Plugin: this.cli.releases.newDeploy is not a function
```

**Comment by kamilogorek** [comment]
Fixed almost a month ago, forgot to release :3 - https://github.com/getsentry/sentry-webpack-plugin/commit/d695dcf433211c7f54e8e984501aace8f75eeaaf

**Comment by kamilogorek** [comment]
Released as `1.12.1`. Cheers!
